src/mibase.ts

```diff
@@ -283,7 +283,11 @@ export class MI2DebugSession extends DebugSession {
 				response.body.threads.push(new Thread(thread.id, thread.id + ":" + threadName));
 			}
 			this.sendResponse(response);
-		}).catch(error => {
+		}).catch((error: MIError) => {
+			if (error.message === 'Selected thread is running.') {
+				console.error(error);
```

Comment:
`console` stuff is not logged on the client, it's just lost

you probably want to either use the DAP error response or send it to the output using `this.handleMsg("stderr", "...")`

Reply:
I can see the `console` when debugging the extension. I wanted to ignore this message because it's not an error.

Reply:
You will have to either do `sendResponse` or `sendErrorResponse` here though, otherwise it's a hanging response that can cause issues with various DAP servers

Reply:
OK, I used `this.handleMsg` like you suggested.